WapoJS/src/host_functions/isolate_eval.rs

```diff
@@ -155,7 +155,10 @@ fn isolate_eval(
     for script in args.scripts {
         output = child_service
             .exec_script(script.as_str())
-            .map_err(Error::msg)?;
+            .map_err(|e| {
+                child_service.close_all();
```

Comment:
This might not be able to solve the issue. The service itself would drop all resources, which is equivalent to `close_all`, before destroying the js engine. Using `child_service.shutdown().await` is better, this require moving the code into an async block (the `fn wait_child`).

Reply:
Could you share the snippet with me? I'm having trouble writing it correctly.